benchmarks/README.md

```diff
@@ -61,11 +61,31 @@ python benchmark_serving.py \
 
 ```
 
+## Benchmark with openorca dataset (openorca is used by MLPerf inference for LLaMA2 models)
+```
+python JetStream/benchmarks/benchmark_serving.py   \
+--tokenizer ~/maxtext/assets/tokenizer.llama2  \
+--warmup-first true   \
+--save-result   \
+--save-request-outputs   \
+--request-outputs-file-path outputs.json   \
+--num-prompts 1000   \
+--max-output-length 1024   \
+--dataset openorca
+
+```
+
 ## Standalone Evaluation Run
 
 If you used `--save-request-outputs`, you can separately evaluate against the generated outputs.
 
 ```
 python eval_accuracy.py
 
-```
\ No newline at end of file
+```
+
+With openorca dataset and llama2-chat models (used by MLPerf), here are the reference accuracy numbers:
+```
+llama2-7b-chat {'rouge1': 42.0706, 'rouge2': 19.8021, 'rougeL': 26.8474, 'rougeLsum': 39.5952, 'gen_len': 1146679, 'gen_num': 998}
```

Comment:
It's nice to see both 7B and 70  accuracy numbers, can you align the columns (adding gen_len and gen_num into 70B row)?

Reply:
i don't have 70B yet. The 70B is from MLPerf. Will put it afterwards after I ran the 70B model myself.